sys/bitfield/bitfield.c

```diff
@@ -46,3 +46,27 @@ int bf_get_unset(uint8_t field[], int size)
     restoreIRQ(state);
     return(result);
 }
+
+int bf_get_first_set(uint8_t field[], int size)
+{
+    int result = -1;
+    int nbytes = (size + 7) / 8;
+    int i = 0;
+
+    unsigned state = disableIRQ();
+
+    /* skip full bytes */
```

Comment:
We had this discussion when @kaspar030 introduced `bf_get_unset` and decided against it. I think it's mostly a question of what sizes for bitfields you expect and trading code space for performance.